### PR TITLE
[RM Ch03]: Change NFVI to Cloud Infrastructure

### LINKED ISSUE #1168: [RM] generic requirements of CNF in RM3.3 and RM4.1
@tomkivlin we have some observation in RM stream . It looks the requirements are too much VNF focused and not scoped adequately for CNF .

E.g 4.1 Capabilities and Performance Measurements we can find VIM performance but not  K8S platform capabilities part 

3.3 NFVI Management Software is another example we expect something like container management as well 
Similarly for K8S at minimum performance requirements , CRD's and CRI-O etc should be included

I am sure it is included somewhere can you ignore my negligence if so and help us understand how you are tacking it .

### PR DESCRIPTION
**Pull request by CsatariGergely**
Partly fixes #1168

### CODE REVIEW

**Comment by rabiabdel** [comment]
@kedmison once you approve, this will get merged.

**Comment by rabi-abdel** [comment]
@kedmison once reviewed/approved by you, it will be merged.

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -177,5 +179,5 @@ Network Function Virtualisation Infrastructure provides the capability to manage
 
 The physical compute, storage and network resources serve as the foundation of the network function virtualisation infrastructure. They are as such not directly exposed to the workloads (VNFs/CNFs).
 
-<p align="center"><img src="../figures/ch03_model_physical_resources.png" alt="NFVI Physical Infrastructure Resources" Title="NFVI Physical Infrastructure Resources" width="65%"/></p>
-<p align="center"><b>Figure 3-4:</b> NFVI Physical Resources.</p>
+<p align="center"><img src="../figures/ch03_model_physical_resources.png" alt="Cloud Infrastructure Physical Infrastructure Resources" Title="Cloud Infrastructure Physical Infrastructure Resources" width="65%"/></p>
```
Comment (kedmison) [inline doc/ref_model/chapters/chapter03.md:182]: The image ch03_model_physical_resources.png contains a reference to "Network Function Virtualization Infrastructure".

Comment (rabi-abdel) [reply doc/ref_model/chapters/chapter03.md:182]: @CsatariGergely do you have the source of the image to fix it?

Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter03.md]: Done

**Comment by rabi-abdel** [comment]
@CsatariGergely I tried to resolve the conflict for you, but since this is from a fork, I was not able to. is it possible to rebase please?

**Pull request merged**